src/scss/_mixins.scss

```diff
@@ -203,6 +211,12 @@
 
 }
 
+@mixin oTeaserWithImage {
```

Comment:
Can these styles be made part of the `oTeaserVideo` mixin, they can be applied using the `o-teaser--has-image` class but like this they are really out of context.